nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -67,6 +67,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   `globalRedirect` can now have redirect codes other than 301 through
   `redirectCode`.
 
+- [](#opt-boot.kernel.sysctl) [`"net.core.wmem_max"`](#opt-boot.kernel.sysctl._net.core.wmem_max_) changed from a string to an integer because of the addition of a custom merge option (taking the highest value defined to avoid conflicts between 2 services trying to set that value), just as [`"net.core.rmem_max"`](#opt-boot.kernel.sysctl._net.core.rmem_max_) since 22.11.
```

Comment:
```suggestion
- [boot.kernel.sysctl."net.core.wmem_max"](#opt-boot.kernel.sysctl._net.core.wmem_max_) changed from a string to an integer because of the addition of a custom merge option (taking the highest value defined to avoid conflicts between 2 services trying to set that value), just as [boot.kernel.sysctl."net.core.rmem_max"](#opt-boot.kernel.sysctl._net.core.rmem_max_) since 22.11.
```